test/deploy/deploy_unittest_py3.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+###
+# usage
+# deploy current version of wmagent external library and use the dmwm/master WMCore code to set up unittest
+# sh ./deploy_unittest.sh
+#
+# Also optional values can be specified. -v deploy agent version, -r git repository for code to test,
+# -b branch name of the repository.
+# following
+# i.e) sh ./deploy_unittest.sh -v 1.0.6 -r ticonaan -v 1.0.5_wmagent
+#
+# for running the test check the tutorial, https://github.com/dmwm/WMCore/wiki/Setup-wmcore-unittest
+###
+DMWM_ARCH=slc7_amd64_gcc630
+VERSION=$(curl -s "http://cmsrep.cern.ch/cgi-bin/repos/comp/$DMWM_ARCH?C=M;O=D" | grep -oP "(?<=>cms\+wmagentpy3-dev\+).*(?=-1-1)" | head -1)
+
+REPOSITORY=dmwm
+BRANCH=
+UPDATE=false
+
+deploy_agent() {
+
+    git clone https://github.com/dmwm/deployment.git
+    curl -s https://raw.githubusercontent.com/dmwm/WMCore/master/test/deploy/init.sh > init.sh
+    
+    # swap the comments on the following lines right before merging
+    curl -s https://raw.githubusercontent.com/mapellidario/WMCore/10487-fix/test/deploy/env_unittest_py3.sh > env_unittest_py3.sh
+    # curl -s https://raw.githubusercontent.com/dmwm/WMCore/master/test/deploy/env_unittest_py3.sh > env_unittest_py3.sh
+    
+    curl -s https://raw.githubusercontent.com/dmwm/WMCore/master/test/deploy/WMAgent_unittest.secrets > WMAgent_unittest.secrets
+    source ./init.sh
+    # set -e
+    for step in prep sw post; do
+        echo -e "\n*** Deploying WMAgent py3: running $step step ***"
+        $PWD/deployment/Deploy -R wmagentpy3-dev@$1    -r comp=comp -t $1 -A $DMWM_ARCH -s $step $INSTALL_DIR wmagentpy3/devtools
```

Comment:
Dario, can you please fix the white spacing in this line?

Reply:
Oh yeah, sure!